src/balena.sbvr

```diff
@@ -716,7 +717,7 @@ Fact type: config has description (Auth)
 
 -- Rules
 
-Rule: It is necessary that each device1 that is managed by a device2, belongs to an application1 that depends on an application2 that owns the device2.
+Rule: It is necessary that each device1 that is managed by a device2, belongs to no applications or belongs to an application1 that depends on an application2 that owns the device2.
```

Comment:
> no applications

Is plural a thing?

Reply:
Yup, you can pluralize things, I think it even handles some edge case plurals as well as the easy case of `s` suffix. It'll work with/without the plural here but reads more naturally with